squarelet/core/views.py

```diff
@@ -19,6 +35,135 @@ class ERHLandingView(TemplateView):
 
     template_name = "core/erh_landing.html"
 
-    def get_context_date(self, **kwargs):
+    def create_search_formula(self, query=None, category=None, provider=None):
+        params = []
+        if not settings.DEBUG:
+            status = '{Status} = "Approved"'
```

Comment:
The correct status is Accepted, not Approved.

```suggestion
            status = '{Status} = "Accepted"'
```